exercises/spiral/spiral_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestSpiral(t *testing.T) {
+	for _, tc := range []struct {
+		x, y int
+		want [][]int
+	}{
+		{1, 1, [][]int{
+			{0},
```

Comment:
I forgot that the output here should be 1, not 0.

Reply:
Are you planning to fix it? :)